src/components/MatrixBuilder/MatrixBuilder.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../assets/styles/variables";
+@import "../../assets/styles/mixins";
```

Comment:
try to import as @import "~assets/styles/mixins"

Reply:
if it will work change it in each styles file

Reply:
no, it doesn`t work

Reply:
ok, may it works only with ts